packages/core/admin/admin/src/pages/ProfilePage/tests/lockedSSO.test.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render, waitFor, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { lightTheme, darkTheme } from '@strapi/design-system';
+import ProfilePage from '../index';
+import serverLockedSSO from './utils/serverLockedSSO';
+import ThemeToggleProvider from '../../../components/ThemeToggleProvider';
+import Theme from '../../../components/Theme';
+
+jest.mock('../../../components/LocalesProvider/useLocalesProvider');
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  useNotification: jest.fn(),
+  useFocusWhenNavigate: jest.fn(),
+  useAppInfo: jest.fn(() => ({ setUserDisplayName: jest.fn() })),
+  useOverlayBlocker: jest.fn(() => ({ lockApp: jest.fn, unlockApp: jest.fn() })),
+}));
+
+const client = new QueryClient({
+  defaultOptions: {
+    queries: {
+      retry: false,
+    },
+  },
+});
+
+const App = (
+  <QueryClientProvider client={client}>
+    <IntlProvider messages={{}} textComponent="span" locale="en">
+      <ThemeToggleProvider themes={{ light: lightTheme, dark: darkTheme }}>
+        <Theme>
+          <ProfilePage />
+        </Theme>
+      </ThemeToggleProvider>
+    </IntlProvider>
+  </QueryClientProvider>
+);
```

Comment:
I'd like to suggest to bring this more in line with the other tests:

```suggestion

const setup = (props) => render(<ProfilePage {...props} />, {
  wrapper({ children ) {
    const client = new QueryClient({
      defaultOptions: {
        queries: {
          retry: false,
        },
      },
    });

    return (
      <QueryClientProvider client={client}>
        <IntlProvider messages={{}} textComponent="span" locale="en">
          <ThemeToggleProvider themes={{ light: lightTheme, dark: darkTheme }}>
            <Theme>
              
            </Theme>
          </ThemeToggleProvider>
        </IntlProvider>
      </QueryClientProvider>
    )
  }
})

```

Which makes it very convenient to use it later on:


```
const { getByText } = setup();
```